config/initializers/instrumentation_notification.rb

```diff
@@ -15,6 +15,8 @@ def process_action(*args)
       raw_payload = {
         controller: self.class.name,
         action: action_name,
+        remote_ip: request.remote_ip,
```

Comment:
looks like `remote_ip` is already present in the payload for API controllers. see app/controllers/concerns/with_payloads.rb. we can try to set `meta` here as well